web/Dockerfile

```diff
@@ -9,7 +9,6 @@ RUN yarn
 
 # Generate map
 RUN mkdir -p public/dist/
-ADD web/build ./build
```

Comment:
I'm curious why this was here 🧐

Reply:
Probably left-over, it works without it :)